server/container/util/src/test/java/org/apache/james/util/StreamUtilsTest.java

```diff
@@ -35,7 +35,7 @@
     public void flattenShouldReturnEmptyWhenEmptyStreams() {
         assertThat(
             StreamUtils.<Integer>flatten(ImmutableList.of())
-                .collect(Guavate.toImmutableList()))
+                .collect(ImmutableList.toImmutableList()))
```

Comment:
Remove all of these collects to rely on stream asserts.

Note that java8 support was added recently in assertj